inc/snowball-enqueue.php

```diff
@@ -1,87 +1,97 @@
 <?php
+global $plugin_path;
+$plugin_path = dirname(__FILE__);
 
 function snowball_add_stylesheets() {
-  echo '<link rel="stylesheet" href="' . plugins_url('snowball/lib/d3-geomap/css/d3.geomap.css') . '">';
-  echo '<link rel="stylesheet" href="' . plugins_url('snowball/lib/fluidbox/css/fluidbox.css') . '">';
-  echo '<link rel="stylesheet" href="' . plugins_url('snowball/lib/font-awesome/css/font-awesome.min.css') . '">';
-  echo '<link rel="stylesheet" href="' . plugins_url('snowball/styles/min/snowball.min.css') . '">';
-  echo '<link rel="stylesheet" href="' . plugins_url('snowball/styles/min/snowball-theme.min.css') . '">';
+  global $plugin_path;
```

Comment:
Can you add an empty line that separates all of the `global $plugin_path;` declarations from the blocks of enqueues/echoes?